src/js/components/molecules/article-image/ArticleImage.vue

```diff
@@ -1,7 +1,14 @@
 <template>
     <figure class="article-image">
         <div class="article-image__container">
-            <c-image class="article-image__image" v-bind="image" v-if="image" resize-behaviour="crop"></c-image>
+            <c-image
+                :sizes="'(max-width: 568px) 100vw, (min-width: 569px, max-width: 767px) 730px, 1080px'"
```

Comment:
Are these breakpoints that we use throughout the website? Is it possible to get these breakpoints and values from some sort of const file? Same for the next line (6)